csrankings-k.csv

```diff
@@ -927,6 +927,7 @@ Kwong-Sak Leung,Chinese University of Hong Kong,http://www.cs.cuhk.edu.hk/~ksleu
 Kyandoghere Kyamakya,University of Klagenfurt,https://www.aau.at/team/kyamakya-kyandoghere,n_5AgzAAAAAJ
 Kyla A. McMullen,University of Florida,https://www.cise.ufl.edu/people/faculty/kyla,DzvXnyoAAAAJ
 Kyle C. Hale,Illinois Institute of Technology,https://science.iit.edu/people/faculty/kyle-hale,zYAOJqwAAAAJ
+Kyle Cranmer,University of Wisconsin - Madison,http://theoryandpractice.org,EZjSxgwAAAAJ

```

Comment:
Web page says "Kyle Cranmer is a Professor of Physics and Data Science at New York University."

Reply:
Kyle doesn't update his personal webpage. Please see his UW-Madison website: https://datascience.wisc.edu/kyle-cranmer/ He is affiliated with Physics, CS, and Statistics departments and the data science institute at UW-Madison since 2022.